src/Sulu/Bundle/ResourceBundle/Resources/public/js/components/condition-selection/main.js

```diff
@@ -449,12 +467,59 @@ define([], function() {
                 case 'simple':
                     return createSimpleInput.call(this, value, constants.valueInputClass);
                 default:
-                    this.sandbox.logger.error('Input type "' + type + '" is not supported!');
-                    break;
+                    return createGenericInput.call(this, value, constants.valueInputClass, operator, parameters);
```

Comment:
i have already opened an issue for that we should be able to manage this additional components in a "proper" way there we could also include more exception handling.